Add allow_other FUSE option to support WSL2/Windows

brush701 opened the pull request:
When running in WSL2 on Windows, the "allow_other" flag is required in order to access the FUSE filesystem from Windows Explorer. With this change, it works as expected on my system:

Edition	Windows 10 Home
Version	20H2
Installed on	‎2/‎10/‎2021
OS build	19042.867
Experience	Windows Feature Experience Pack 120.2212.551.0

rschroll commented:
Thanks for the investigation and the fix.  I was not expecting this to work at all on Windows!

Before merging this in, two items:
1. While I suspect that rmfuse isn't going to be used on any truly multi-users systems, I still feel like `allow_other` should be off by default, if possible.  Two ideas I have:
    a. Enable `allow_other` only on Windows.
    b. Add a flag for `allow_other`.
2. Do you mind adding instructions for other Windows users to the README?

Thanks again, and sorry it took me so long to get to this.  (Life, you know.)

brush701 commented:
Makes sense! While I was implementing these changes, I noticed that I've started getting 401 errors on all of the 3rd party clients. Have you seen this behavior? I tried resetting connection with new tokens, but now it looks like that endpoint is returning HTML rather than the tokens themselves. Not sure if this is a problem with my config or a more substantial change on their end.

rschroll commented:
Yeah, they changed a URL in the auth process.  See #23 for details.

brush701 commented:
Gotcha. Should be set now. I updated `poetry.lock` to pull in the latest rmcl to update the URL endpoint, added a flag for `allow_other`, and updated the readme.

rschroll commented:
Thanks!  I reverted the changes to the lock file, since the updated rmcl should still work with the older versions.  Let me know if this isn't the case for you.
